packages/frontend/src/redux/slices/flowLimitation/index.js

```diff
@@ -21,11 +22,18 @@ const initialState = {
 const handleFlowLimitation = createAsyncThunk(
     `${SLICE_NAME}/handleFlowLimitation`,
     async (_, thunkAPI) => {
+        const { actions: { setFlowLimitation } } = flowLimitationSlice;
         const { dispatch, getState } = thunkAPI;
         const { pathname } = getState().router.location;
-        const { redirect_url } = getState().account.url;
-        const { actions: { setFlowLimitation } } = flowLimitationSlice;
 
+        if (WALLET_CREATE_NEW_ACCOUNT_FLOW_URLS.some((url) => pathname.includes(url))) {
```

Comment:
This covers 10 URLs, are we pretty confident that there are no valid use cases for switching accounts in any of them? In particular I'm wondering about those involved with funding an account, where a user might want to switch to the funding account to send funds to the new account. Would this affect that flow at all?

Reply:
Yep any pages where switching account would be relevant, like funding pages, have an account selector on the page 👍